cpp/include/cudf/detail/transpose.hpp

```diff
@@ -30,9 +30,10 @@ namespace detail {
  * @param[in] input Input table of (ncols) number of columns each of size (nrows)
  * @return Newly allocated output table with (nrows) columns each of size (ncols)
```

Comment:
Doc update

Reply:
Updated. Does the new doc string make sense?